### PR TITLE
feat: cache manifest for one day

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
I think that's long enough to keep the number of requests acceptably low
If we forget to change this before updating the manifest, it's also not
too long a period to have, say, broken icons.

Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

**Review by ShaunSHamilton — APPROVED** [review]
Should do something 👍 

---

Clarify for me, though: The client is in charge of keeping track of the resources age? So, 1_000_000 unique users per day is still 1_000_000 requests the vms must handle per day?

**Comment by ojeytonwilliams** [comment]
> The client is in charge of keeping track of the resources age?

Nope, that's the responsibility of the browser and any caches between it and the origin.

For simplicity, imagine the only caches are the CDN (Cloudflare) and the browser. Now imagine someone visits the page for the first time (in a given browser). The browser won't have anything in its cache and so will make a request. The CDN _may_ have the resource cached. If it does, it'll check the Cache-Control header and determine if the resource is fresh or not. If it is, then it sends it back to the browser. If not, it queries the origin (i.e the step we're trying to minimise).

Now, if the same user reloads their page. Their browser will look at the header, check the max-age, see that this is fresh and in its cache. It will simply return what it has in the cache.

At this point another user may visit the page for the first time and the CDN will definitely still have the resource and so doesn't bother the origin.

TL;DR: browser + CDN check max-age and this should result in only one request reaching the origin per day.

**Review by naomi-lgbt — APPROVED** [review]
